packages/flutter_tools/lib/src/android/android_device_discovery.dart

```diff
@@ -96,6 +106,12 @@ class AndroidDevices extends PollingDeviceDiscovery {
     String text, {
     List<AndroidDevice> devices,
     List<String> diagnostics,
+    AndroidSdk androidSdk,
+    FileSystem fileSystem,
+    Logger logger,
+    Platform platform,
+    ProcessManager processManager,
+    @required TimeoutConfiguration timeoutConfiguration,
```

Comment:
I forgot to make this required, and it was null - causing the failure.

Reply:
How did that pass the Cirrus tests the first time?

Reply:
I guess no integration test hit android device discovery for real.

Reply:
FWIW I did test the change initially, but not after I updated device discovery 🤦